src/dev-app/system-rxjs-operators.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
```

Comment:
Should we have this file on a higher level than the `dev-app`? It could be useful for some of the other apps.

Reply:
If we need it for other apps, we could always still move it up. Right now it's just the dev-app, and I won't switch the tests to the UMD bundles because Bazel is automatically concating the test JS files, so it's probably not worth the effort.